classes/submission/reviewAssignment/ReviewAssignmentDAO.inc.php

```diff
@@ -376,15 +375,21 @@ function insertObject($reviewAssignment) {
 		);
 
 		$reviewAssignment->setId($this->getInsertId());
-		return $reviewAssignment->getId();
+
+		// Update review stage status whenever a review assignment is changed
+		if ($result) {
+			$this->updateReviewRoundStatus($reviewAssignment);
+		}
+
+		return $reviewAssignment;
```

Comment:
OK. Have you grepped around for the old return expectation? If there's a pattern you'd like to introduce for returns from insert/update functions, feel free to propose it. It would be nice to get everyone working towards standardization on this kind of thing.